services/web/server/src/simcore_service_webserver/notifications/plugin.py

```diff
@@ -30,3 +33,4 @@ def setup_notifications(app: web.Application):
     # Subscribe to rabbit upon startup for logs, progress and other
     # metrics on the execution reported by sidecars
     app.cleanup_ctx.append(setup_rabbitmq_consumers)
+    app.cleanup_ctx.append(setup_rabbitmq_osparc_credits_consumers)
```

Comment:
MINOR: if possible use other name for the callback ... otherwise we confuse it with the setup_plugin function

Reply:
Sorry, I do not fully understand what are you proposing? should I change `setup_rabbitmq_consumers` & `setup_rabbitmq_osparc_credits_consumers` names?